Jenkinsfile

```diff
@@ -0,0 +1,12 @@
+properties([
+    durabilityHint('PERFORMANCE_OPTIMIZED'),
+    buildDiscarder(logRotator(numToKeepStr: '5')),
+])
+    
+node ("docker") {
+  checkout scm
+  sh "make docker"
+  sh "make run"
+  // TODO: Update te demo to support using the Azure mirror instead of the local Maven volume
```

Comment:
Do you prefer using two commands instead of "make docker run" by any reason? By the way there is a typo in "the demo" :P

Reply:
> Do you prefer using two commands instead of "make docker run" by any reason?

I want to switch to stages later, I was just too lazy for it :)

Reply:
done